### PR TITLE
Add disconnected annotation to csv

### PR DESCRIPTION
**Pull request by rubenvp8510**
Signed-off-by: Ruben Vargas <ruben.vp8510@gmail.com>

### CODE REVIEW

File: deploy/olm-catalog/jaeger-operator/manifests/jaeger-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -23,6 +23,7 @@ metadata:
     support: Jaeger Community
   name: jaeger-operator.v1.25.0
   namespace: placeholder
+  operators.openshift.io/infrastructure-features: '["disconnected"]'
```
Comment (pavolloffay) [inline deploy/olm-catalog/jaeger-operator/manifests/jaeger-operator.clusterserviceversion.yaml:26]: What does this exactly mean? that the operator can work in a cluster without internet connectivity? 

I have found only this https://docs.openshift.com/container-platform/4.5/operators/operator_sdk/osdk-generating-csvs.html

**Review by pavolloffay — APPROVED** [review]

File: deploy/olm-catalog/jaeger-operator/manifests/jaeger-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (pavolloffay) [reply deploy/olm-catalog/jaeger-operator/manifests/jaeger-operator.clusterserviceversion.yaml:26]: https://docs.openshift.com/container-platform/4.6/operators/operator_sdk/osdk-generating-csvs.html

**Comment by pavolloffay** [comment]
@rubenvp8510 why is this change required?

File: deploy/olm-catalog/jaeger-operator/manifests/jaeger-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (jpkrohling) [reply deploy/olm-catalog/jaeger-operator/manifests/jaeger-operator.clusterserviceversion.yaml:26]: Yes -- this means that the operator can run on a private network without access to the public internet.

Comment (jpkrohling) [reply deploy/olm-catalog/jaeger-operator/manifests/jaeger-operator.clusterserviceversion.yaml:26]: This annotation seems to be misplaced: it's under metadata, should be under annotations.

**Review by jpkrohling — COMMENTED** [review]
Needs to fix the annotation placement.

**Comment by rubenvp8510** [comment]
> Needs to fix the annotation placement.

Ooops, yes, it should be under annotation node, I've already fixed this. 

Thanks

**Review by jpkrohling — APPROVED** [review]

**Pull request merged**